pyomo/contrib/pynumero/tests/test_cyipopt_examples.py

```diff
@@ -13,6 +13,10 @@
 import pyomo.common.unittest as unittest
 import pyomo.environ as pyo
 from pyomo.common.dependencies import attempt_import
+from pyomo.common.log import LoggingIntercept
+from pyomo.opt import TerminationCondition
+from six import StringIO
```

Comment:
As we no longer support Python 2.7, we are removing the use of `six`.  Recommend replacing with
```python
from io import StringIO
```

Reply:
done.